content/cpp/concepts/sets/terms/erase/erase.md

```diff
@@ -0,0 +1,54 @@
+---
+Title: '.erase()'
+Description: 'Removes a single value from a set.'
+Subjects:
+  - 'Computer Science'
+  - 'Game Development'
+Tags:
+  - 'Data Types'
+  - 'Sets'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The `.erase()` method removes a single value from a set.
+
+## Syntax
+
+```pseudo
+setName.erase(value);
+```
+
+- `value` is the element to be removed from the set.
+
+## Codebyte Example
+
+The following example uses `.erase()` to remove elements from the `stoneFruits` set:
+
+```codebyte/cpp
+#include <iostream>
+#include <set>
+#include <string>
+
+int main(){
+  // Initiate set
+  std::set<std::string> stoneFruits;
+
+  // Insert values into set
+  stoneFruits.insert("Apricots");
+  stoneFruits.insert("Peaches");
+  stoneFruits.insert("Nectarines");
+  stoneFruits.insert("Plums");
+
+  // Remove values from set
+  stoneFruits.erase("Apricots");
+  stoneFruits.erase("Nectarines");
+
+  // Print out set
+  std::set<std::string> :: iterator iter;
+  for(iter = stoneFruits.begin(); iter != stoneFruits.end(); iter++){
+    std::cout<< *iter << "\n";
+  }
+}
+```
```

Comment:
You have an output example for `.insert()` but not here?

Reply:
@SSwiniarski Yes, @Dusch4593 recommended I change the `.insert()` example to a codebyte example in the previous review and I forgot to remove it when making the changes. I'll take it out now, thanks!